deploy/operator/utils.sh

```diff
@@ -48,8 +48,3 @@ function wait_for_pod() {
   echo "Waiting for pod (${pod}) on namespace (${namespace}) with labels (${selector}) to become ready..."
   oc wait -n "$namespace" --for=condition=Ready pod --selector "$selector" --timeout=10m
 }
-
-function print_help() {
-  ALL_FUNCS=$(compgen -A "function" | grep -Ev "(help|wait_for)" | paste -s -d'|')
-  echo "Usage: DISKS=\$(echo sd{b..f}) bash ${0} (${ALL_FUNCS})"
-}
```

Comment:
shared ``print_help`` function doesn't seem to be a good idea, so I separated it between different scripts

Reply:
Why?

Reply:
Because ``DISCONNECTED`` doesn't make sense in the context of disks creation